### PR TITLE
Adding allowedUsers and allowedGroups documentation for printerShare

### PR DESCRIPTION
**Pull request by braedenp-msft**
Collections for allowedUsers and allowedGroups are being added to the `printerShare` entity.

Updates:
* Added navigation properties to the `printerShare` entity
* Added API pages for listing, creating and deleting allowedUsers and allowedGroups entries
* Updated changelog.md

### CODE REVIEW

**Comment by Lauragra** [comment]
@braedenp-msft , what is the merge status of this PR? I'm setting to do no merge; please set *ready to merge* when ready. Also, you'll want to add the new method topics to the TOC; please update the TOC file.

**Comment by braedenp-msft** [comment]
> @braedenp-msft , what is the merge status of this PR? I'm setting to do no merge; please set _ready to merge_ when ready. Also, you'll want to add the new method topics to the TOC; please update the TOC file.

Thanks Laura! The PR is now ready to merge after you sign off (updated the TOC as well).

File: api-reference/beta/api/printershare-list-allowedgroups.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+|:---------------|:--------------------------------------------|
+|Delegated (work or school account)| Users.Read.All |
+|Delegated (personal Microsoft account)|Not Supported.|
+|Application|Not Supported.|
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET /print/printerShares/{id}/allowedGroups
+```
```
Comment (Lauragra) [inline api-reference/beta/api/printershare-list-allowedgroups.md:33]: List topics should include an "Optional query parameters" H2. Ideally that section mentions what query parameters are supported to customize the response. Or we can use the generic boilerplate text,

Comment (braedenp-msft) [reply api-reference/beta/api/printershare-list-allowedgroups.md:33]: Looks like we don't have that in any of the Universal Print docs. Should I add that for the files added in this PR, or should I add to all files in a separate PR after this is merged?

**Comment by Lauragra** [comment]
@braedenp-msft , I see the Optional query parameters section in most of the List and Get topics in the Print node. Do you want to go ahead and merge this and you can do a separate PR to add it to these and any others that are missing?

File: api-reference/beta/api/printershare-list-allowedgroups.md (hunk 1 above)
Comment (Lauragra) [reply api-reference/beta/api/printershare-list-allowedgroups.md:33]: I see the Optional query parameters section in most of the List and Get topics in the Print node. Do you want to go ahead and merge this and you can do a separate PR to add it to these and any others that are missing?

**Comment by braedenp-msft** [comment]
> @braedenp-msft , I see the Optional query parameters section in most of the List and Get topics in the Print node. Do you want to go ahead and merge this and you can do a separate PR to add it to these and any others that are missing?

Oh that's right! Just missing for the topics that aren't List and Get.

Yes, can you please merge this PR and I'll create a new one tomorrow with the Query options updates?

**Comment by Lauragra** [comment]
> > @braedenp-msft , I see the Optional query parameters section in most of the List and Get topics in the Print node. Do you want to go ahead and merge this and you can do a separate PR to add it to these and any others that are missing?
> 
> Oh that's right! Just missing for the topics that aren't List and Get.
> 
> Yes, can you please merge this PR and I'll create a new one tomorrow with the Query options updates for the missing pages?

Yes, I'll go ahead and merge. FYI, we only need that section for List and Get APIs that return one or more objects.

**Pull request merged**